data/movements/scripts/citizen.lua

```diff
@@ -1,12 +1,11 @@
 function onStepIn(creature, item, position, fromPosition)
 	if item.actionid > 30020 and item.actionid < 30050 then
 		local player = creature:getPlayer()
-		if player == nil then
+		if not player then
```

Comment:
You can use `creature:isPlayer()` and use `creature` downwards assuming it has `Player` metatable. On lots of functions, actually.